cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -286,6 +286,71 @@
           "description": "A description of the methodology used for the production of this data/software",
           "type": "string"
         },
+        "steps": {
+          "items":[
+            {
+              "properties": {
+                "configuration_files": {
+                  "items": [
+                    {
+                      "properties": {
+                        "script": {
+                          "description": "A cmsdriver script",
+                          "type":"string"
+                        },
```

Comment:
As this will contain the full production script including all commands for the job that was run for production (with cmsrel .... , cd ...., and then cmsdriver [parameters]), maybe we could call it "Production script".

Reply:
Also, sometimes it will contain genfragment blobs, not only cmsdriver blobs, so we'd need a rather generic description.